### PR TITLE
[Python] Simulation and other trivial improvements

### PR DESCRIPTION
**Pull request by davidiw**
* Cleaned up some typing
* Some aesthetic changes
* Support for simulating txns within python
* Basic support for a client config

tested locally and now with ci/cd!

### CODE REVIEW

File: ecosystem/python/sdk/examples/simulate-transfer-coin.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    print("\n=== Simulate before creatng Bob's Account ===")
+    output = rest_client.simulate_transaction(transaction, alice)
+    assert output[0]["vm_status"] != "Executed successfully", "This shouldn't succeed"
+    print(json.dumps(output, indent=4, sort_keys=True))
+
+    print("\n=== Simulate after creatng Bob's Account ===")
+    faucet_client.fund_account(bob.address(), 0)
+    output = rest_client.simulate_transaction(transaction, alice)
+    assert output[0]["vm_status"] == "Executed successfully", "This should succeed"
+    print(json.dumps(output, indent=4, sort_keys=True))
```
Comment (gregnazario) [inline ecosystem/python/sdk/examples/simulate-transfer-coin.py:48]: I guess this is the test?

**Review by gregnazario — COMMENTED** [review]
It looks fine to me, except we really need a test harness around this code

**Review by gregnazario — APPROVED** [review]

File: ecosystem/python/sdk/examples/simulate-transfer-coin.py (hunk 1 above)
Comment (davidiw) [reply ecosystem/python/sdk/examples/simulate-transfer-coin.py:48]: yes...

**Comment by davidiw** [comment]
> It looks fine to me, except we really need a test harness around this code

I'm working on a set of new platform tasks. Some good stuff for some eager folks in the ecosystem :)

**Review by BIGTECH1982 — COMMENTED** [review]
Thank You GUYS! Willie Parish IPHONE 12 PRO MAX

**Review by BIGTECH1982 — COMMENTED** [review]
THANK YOU GUYS! WILLIE PARISH IPHONE 12 PRO MAX

**Pull request merged**